_pages/leaderboard.md

```diff
@@ -11,9 +11,9 @@ permalink: /leaderboard/
 | Ben           | 3             | 7          |
 | Bernard       | 0             | 1          |
 | Vikram        | 1             | 4          |
-| Kirin         | 2             | 1          |
+| Kirin         | 2             | 4          |
 | Josh          | 0             | 1          |
 | Evan          | 0             | 2          |
-| Drew          | 1             | 1          |
+| Drew          | 2             | 1          |
 | Erick         | 2             | 0          |
-| Aditya        | 0             | 1          |
+| Aditya        | 2             | 1          |
```

Comment:
I don't think we're doing 1L as 3 ices
```suggestion
| Aditya        | 1             | 1          |
```

Reply:
I feel like a 1L should be worth more tho

Reply:
It's been worth 1 historically, if we change it for this one we'd have to change for all the past ones too